test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
For some reason test.sh appears again in the PR diff. Please remove.

Reply:
DOne